lib/Zonemaster/Backend/Config.pm

```diff
@@ -216,72 +224,109 @@ sub check_db {
     }
 }
 
-sub BackendDBType {
-    my ($self) = @_;
-    return $self->DB_engine;
-}
+
+=head2 DB_engine
+
+Get the value of L<DB.engine|https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Configuration.md#polling_interval>.
```

Comment:
It should not be "polling_interval" here.

Reply:
Good catch! I fixed it.